packages/backend-common/config.d.ts

```diff
@@ -57,7 +57,7 @@ export interface Config {
     database:
       | {
           client: 'sqlite3';
-          connection: ':memory:' | string;
+          connection: ':memory:' | string | { filename: string };
```

Comment:
The tests were testing for `{filename: "xxx"}` but code also supports `"xxx"` so I extended the schema